### PR TITLE
Wordpress Fetch/Update User Action

### LINKED ISSUE #5335: [Action] Wordpress Fetch/Update User Specifically User Meta
Wordpress.org

Fetch User and Update User https://developer.wordpress.org/rest-api/reference/users/#update-a-user-2

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #5335

### CODE REVIEW

File: components/wordpress_org/actions/get-user/get-user.mjs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import wordpress from "../../wordpress_org.app.mjs";
+
+export default {
+  key: "wordpress_org-get-user",
+  name: "Get User",
+  description: "Retrieves information for a user. [See the docs here](https://developer.wordpress.org/rest-api/reference/users/#update-a-user-2)",
```
Comment (feyzullah) [inline components/wordpress_org/actions/get-user/get-user.mjs:6]: doc link might not be correct. Can you please check the link?
```suggestion
  description: "Retrieves information for a user. [See the docs here](https://developer.wordpress.org/rest-api/reference/users/#retrieve-a-user)",
```

**Review by feyzullah — COMMENTED** [review]
Hi @michelle0927 I left only one comment. Can you please check?

**Review by feyzullah — DISMISSED** [review]

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Wordpress_5335_1346.pdf](https://github.com/PipedreamHQ/pipedream/files/10740073/Wordpress_5335_1346.pdf)

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Wordpress_5335_1053.pdf](https://github.com/PipedreamHQ/pipedream/files/10751988/Wordpress_5335_1053.pdf)

**Review by feyzullah — APPROVED** [review]

**Pull request merged**